microverse_collins_yemi-main/fcc-test-suite-template-master/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <div class="main">
+    <div>
+      <h1>Handcrafted, home-made masterpieces</h1>
+      <input placeholder="Enter your email">
+      <h1 class="bgYellow">SELECT</h1>
+    </div>
+
+    <div>
+      <h1>Premium Materials</h1>
+      <p>Our trombones use the shiniest brass which is sourced locally. This will increase the longevity of your purchase.</p>
+    </div>
+
+    <div>
+      <h1>Fast Shipping</h1>
+      <p>We make sure you recieve your trombone as soon as we have finished making it. We also provide free returns if you are not satisfied.</p>
+    </div>
+
+    <div>
+      <h1>Quality Assurance</h1>
+      <p>For every purchase you make, we will ensure there are no damages or faults and we will check and test the pitch of your instrument.</p>
+    </div>
+
+  </div>
```

Comment:
Please consider replacing all div tags with the section tag to make it semantically correct and use other hx tags as well in this section